Godeps/_workspace/src/github.com/docker/docker/pkg/reexec/command_solaris.go

```diff
@@ -0,0 +1,21 @@
+package reexec
```

Comment:
It seems this dependeny file and the next one were added manually, instead of via the godep tool.